raml-parser-interface/src/main/java/org/mule/apikit/model/parameter/Parameter.java

```diff
@@ -18,6 +19,15 @@ public interface Parameter {
 
   String getDefaultValue();
 
+  /**
+   * Returns the list of default values.
+   * <p>
+   * Empty list if no default
+   *
+   * @since X.Y.Z (TBD)
```

Comment:
Version TBD

Reply:
We should `@Deprecate String getDefaultValue()`